test/aura/AuraAvatarMultiTokenTest.t.sol

```diff
@@ -47,76 +61,66 @@ contract AuraAvatarTwoTokenTest is Test, AuraAvatarUtils {
     event TwapPeriodUpdated(uint256 newTwapPeriod, uint256 oldTwapPeriod);
     event ClaimFrequencyUpdated(uint256 oldClaimFrequency, uint256 newClaimFrequency);
 
-    event SellBpsBalToUsdcUpdated(uint256 oldValue, uint256 newValue);
-    event SellBpsAuraToUsdcUpdated(uint256 oldValue, uint256 newValue);
+    event SellBpsAuraToUsdcUpdated(uint256 newValue, uint256 oldValue);
 
     event MinOutBpsBalToUsdcMinUpdated(uint256 oldValue, uint256 newValue);
     event MinOutBpsAuraToUsdcMinUpdated(uint256 oldValue, uint256 newValue);
-    event MinOutBpsBalToBptMinUpdated(uint256 oldValue, uint256 newValue);
 
     event MinOutBpsBalToUsdcValUpdated(uint256 oldValue, uint256 newValue);
     event MinOutBpsAuraToUsdcValUpdated(uint256 oldValue, uint256 newValue);
-    event MinOutBpsBalToBptValUpdated(uint256 oldValue, uint256 newValue);
 
     event Deposit(address indexed token, uint256 amount, uint256 timestamp);
     event Withdraw(address indexed token, uint256 amount, uint256 timestamp);
 
     event RewardClaimed(address indexed token, uint256 amount, uint256 timestamp);
     event RewardsToStable(address indexed token, uint256 amount, uint256 timestamp);
 
-    event Transfer(address indexed from, address indexed to, uint256 value);
-
     function setUp() public {
         // TODO: Remove hardcoded block
-        vm.createSelectFork("mainnet", 15858000);
+        vm.createSelectFork("mainnet", 16221000);
 
         // Labels
         vm.label(address(AURA), "AURA");
         vm.label(address(BAL), "BAL");
         vm.label(address(WETH), "WETH");
         vm.label(address(USDC), "USDC");
-        vm.label(address(AURABAL), "AURABAL");
-        vm.label(address(BPT_80BAL_20WETH), "BPT_80BAL_20WETH");
 
-        avatar = new AuraAvatarTwoToken(
-            PID_80BADGER_20WBTC,
-            PID_40WBTC_40DIGG_20GRAVIAURA
-        );
-        avatar.initialize(owner, manager, keeper);
+        uint256[] memory pidsInit = new uint256[](PIDS.length);
+        pidsInit[0] = PIDS[0];
+        pidsInit[1] = PIDS[1];
+        pidsInit[2] = PIDS[2];
+        avatar = new AuraAvatarMultiToken();
+        avatar.initialize(owner, manager, pidsInit);
 
-        deal(address(avatar.asset1()), owner, 10e18, true);
-        deal(address(avatar.asset2()), owner, 20e18, true);
+        for (uint256 i; i < PIDS.length; ++i) {
+            deal(address(BPTS[i]), owner, 20e18, true);
 
-        vm.startPrank(owner);
-        BPT_80BADGER_20WBTC.approve(address(avatar), 10e18);
-        BPT_40WBTC_40DIGG_20GRAVIAURA.approve(address(avatar), 20e18);
-        vm.stopPrank();
+            vm.prank(owner);
+            BPTS[i].approve(address(avatar), 20e18);
+        }
     }
 
     ////////////////////////////////////////////////////////////////////////////
     // Initialization
     ////////////////////////////////////////////////////////////////////////////
 
     function test_constructor() public {
-        assertEq(avatar.pid1(), PID_80BADGER_20WBTC);
```

Comment:
nit but given that there is no constructor anymore maybe we should group all of this within the init test

Reply:
resolved in ed2ffa5b2116d7788bd4c529a75692aa7615deba